apps/condo/domains/organization/integrations/sbbol/sync/syncBankAccounts.js

```diff
@@ -0,0 +1,100 @@
+const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { ISO_CODES_FOR_SBBOL } = require('@condo/domains/common/constants/currencies')
+const { dvSenderFields } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { initSbbolFintechApi } = require('@condo/domains/organization/integrations/sbbol/SbbolFintechApi')
+
+
+const logger = getLogger('sbbol/syncBankAccounts')
+
+/**
+ * Connects new BankAccount records for user according to accounts data from SBBOL.
+ *  @param {SbbolAccount} accounts
+ *  @param {Object} organization
+ */
+const _syncBankAccounts = async (accounts, organization) => {
+    const { keystone: context } = await getSchemaCtx('User')
+
+    for (const account of accounts) {
+        const bankAccountDetails = {
+            tin: organization.tin,
+            country: RUSSIA_COUNTRY,
+            number: account.number,
+            currencyCode: (ISO_CODES_FOR_SBBOL[account.currencyCode]),
+            routingNumber: account.bic,
+        }
+        const foundAccount = await BankAccount.getOne(context, {
+            ...bankAccountDetails,
+            organization: { id: organization.id },
+        })
+
+        if (!foundAccount) {
+            const integration = await BankIntegration.getOne(context, { id: BANK_INTEGRATION_IDS.SBBOL })
+            if (!integration) throw new Error(`Cannot find SBBOL integration by id=" ${BANK_INTEGRATION_IDS.SBBOL}"`)
+
+            const bankIntegrationContext = await BankIntegrationContext.create(context, {
+                ...dvSenderFields,
+                integration: { connect: { id: integration.id } },
+                organization: { connect: { id: organization.id } },
+            })
+
+            await BankAccount.create(
+                context,
+                {
+                    meta: {
+                        sbbol: {
+                            type: account.type,
+                            state: account.state,
+                            openDate: account.openDate,
+                            name: account.name,
+                            closeDate: account.closeDate,
+                        },
+                    },
+                    ...bankAccountDetails,
+                    ...dvSenderFields,
+                    integrationContext: { connect: { id: bankIntegrationContext.id } },
+                    organization: { connect: { id: organization.id } },
+                }
+            )
+            logger.info('Created BankAccount', { bankAccount: { id: account.number, organization: { id: organization.id, name: organization.name } } })
+        }
+    }
+}
+/**
+ * Fetches ClientAccounts from SBBOL for specified date and creates BankAccount if this bank account has not yet been created
+ *
+ *  @param {String} userId
+ *  @param {Object} organization
+ * @return {Promise<void>}
+ */
+const syncBankAccounts = async (userId, organization) => {
```

Comment:
You already had function with name like this. And also it has a `private` marker. But this function could be used anywhere, because of you are exporting it. May it be renamed it to something different?

Reply:
Oh, I just saw that this private function used at tests. May 'public' function replace it?

Reply:
I wrote this when I didn't know how to use jest.mock so to avoid using a client that uses an external API, I just used only the data persistence part. I can rewrite it but it will take time

Reply:
And it seems later the tests will not work as everything will be transferred to an external service. That's probably why it's not worth it.